### ISSUE TITLE
AppImage for most Linux distributions

### ISSUE THREAD

**Issue by probonopd** [geometer/FBReader#288]
I have created an experimental [AppImage](http://appimage.org/) of FBReader 0.99.4:
https://bintray.com/probono/AppImages/FBReader/view#files

Just download, mark as executable, and run. Tested on Xubuntu 16.04 and CentOS 7, but should run on most not-too-outdated distributions from Ubuntu trusty and up. If it doesn't, then the recipe will need a little bit more fine-tuning.

[Here](https://github.com/probonopd/AppImages/blob/master/recipes/fbreader/Recipe) is the script that was used to generate the AppImage; however it would be preferred if the upstream project would provide "official" AppImages built by the project as Krita, MuseScore, and Scribus already do. Distribute your desktop Linux application in the AppImage format and win users running all common Linux distributions. Package once and run everywhere. Reach users on all major desktop distributions.

**Comment by haevalencia** [comment]
There are ways to distribute software more promising as Snap or Flatpak, however, it seems the development team is more interested in other [things](https://github.com/geometer/FBReader/issues/285)

**Comment by probonopd** [comment]
@haevalencia AppImage is by far the easiest of the bundling solutions, much, much simpler than the others, and not slanted toward any particular distribution.

**Comment by haevalencia** [comment]
@probonopd  Yes it is simpler and less functionality by design (can be used bubblewrapp for sandbox, etc.). 
Only Snap is inclined to a particular distribution (Ubuntu), but is now Universal. Flatpak was universal from the beginning so I do not understand much your claim.

Developers will decide in the future, it is now a *.deb + tarball.

**Comment by probonopd** [comment]
@haevalencia Snap is installed by default only on Ubuntu and Flatpak is installed by default only on Fedora. AppImage works by default on both, and on many more and older distributions, too.

**Comment by haevalencia** [comment]
@probonopd Wow ... today everything is installed easy with a few clicks. Things will change in the future but... AppImage by default is not sandboxed.

**Comment by idomeneo** [comment]
I'm having trouble opening files with this Appimage - even if I put quotes around the file name, the app (or possibly fbreader.wrapper) seems to remove them, so that if a filename/book title has any spaces in it, each word gets taken for a separate argument, and FBReader cannot open the file.
